packages/engine-server/src/workspace/service.ts

```diff
@@ -1504,4 +1512,15 @@ export class WorkspaceService implements Disposable, IWorkspaceService {
       },
     });
   }
+
+  resetMeta() {
```

Comment:
can we also add a flag indicating that the metadata has been reset? can be a new property that we should also report on

Reply:
I've added this so I can make `ResetConfigCommand` to clean this up, but I ended up keeping that command intact for now. (so this method is not used anywhere yet. I might just remove it)

I think it serves very little use cases for the users and only for dev purposes.

Even then, since we are now dealing with a simple json files (`.dendron.ws` and `meta.json`) instead of a sqlite database (vscode's `globalStorage` and `workspaceStorage`), I think it's better to directly modify / backup the actual files rather than have a command that nukes it.

At the very least we should have the enablement set to false in production. What do you think? (analytics show that this command has very little use. I suspect they are not intentional uses as well)

Reply:
also, most internal use of the command were in tests (and I suspect it will be moving forward as well), and now this being tied to a workspace service kinda makes it useless for those cases. We can't use it in hooks that run before workspace initialization. I would probably opt for creating a utility function for tests that directly grabs the files filesystem and modifies them.

I'll remove them from the workspace service until we find the need to reintroduce them

Reply:
for a separate PR, can we introduce a new flag? (eg. enableDevContext). if set, we switch to development context which enables developer only commands like resetConfig